internal/buildmanifest/manifest.go

```diff
@@ -0,0 +1,66 @@
+package buildmanifest
+
+import "gopkg.in/yaml.v2"
+
+const (
+	// Filename of a build manifest.
+	Filename = "build.yml"
+	// CurrentVersion os the manifest file format. See Manifest.SDK.
+	CurrentVersion = "2"
+)
+
+// Artifact is a file copied from one path to another during the build.
+type Artifact struct {
+	Path string `yaml:"path"`
+	Dest string `yaml:"dest"`
+}
+
+// Manifest used for the declarative build system for drivers.
+type Manifest struct {
+	// SDK version. Used to track future changes to this file. Current version is '2'.
+	SDK string `yaml:"sdk"`
+
+	Native struct {
+		// Image is a Docker image used as the native driver runtime.
+		Image string `yaml:"image"`
+		// Static is a list of files that will be copied from native driver's source directory
+		// to the final driver image. Note that those files cannot be modified by the build.
+		Static []Artifact `yaml:"static"`
+		// Deps is a list of apt/apk commands executed in the final driver image.
+		// This directive should be avoided since a different Docker image may be used instead of it.
+		Deps  []string `yaml:"deps"`
+		Build struct {
+			// Gopath directive sets a new gopath for the driver build. Only used by Go drivers.
```

Comment:
Can you please mention the format of this value? Is it a single directory, or a `a:b:c` style path list?

Reply:
(Even if it's just "whatever you would poke into `GOPATH`", it would be helpful to say that in the comment)